app/styles/component/_post-search-results-item.scss

```diff
@@ -0,0 +1,46 @@
+.post-search-results-item {
+  margin: 18px 0;
+
+  // Clearfix
+  &::after {
+    clear: both;
+    content: "";
+    display: block;
+    height: 0;
+    visibility: hidden;
+  }
+
+  &__title {
+    color: #0d7675;
```

Comment:
is it color not defined in design system? Out of curiosity, can you send me link to designs?

Reply:
https://zpl.io/aRzpBkn

Reply:
Actually it's awfully close to `$wds-fandom-color-link`, going to use that instead.

Reply:
This module will go though Design Review after it's merged, so I think we're OK with current colors.